extpreimage/client.go

```diff
@@ -107,7 +111,71 @@ func (c *client) Retrieve(req *GetPreimageRequest) (*GetPreimageResponse,
 	}
 }
 
-// Stop closes any outstanding grpc connections to allow for a graceful shutdown
+// symbol converts the configured chain to a symbol to differentiate between
+// currencies in the format that ExternalPreimage service expects.
+// LND keeps track based on chainCode, but, as that value is private, we use
+// its string representation to do our conversion.
+func (c *client) symbol() (Symbol, error) {
+	var symbol Symbol
```

Comment:
[Nit] instead of using a variable, could we just return from each condition? (Im not sure if this is a golang thing or not) Example:

```
if c.chain == "bitcoin" {
  return Symbol_BTC, nil
} ....
```